image_tools/README.md

```diff
@@ -0,0 +1,43 @@
+## **What Is This?**
+
+This demo provides a simple utility to **connect to a workstation default camera device** and **display** it in an **OpenCV** window like below:
+
+![](img/result.png)
+
+## **Build**
+
+```bash
+cd $HOME
+mkdir -p demo_ws/src
+cd ~/demo_ws/src
+git clone https://github.com/ros2/demos.git
+cd ~/demo_ws
+source /opt/ros/<DISTRO_NAME>/setup.bash
+# source /opt/ros/rolling/setup.bash
+colcon build --package-select image_tools
+```
+
+## **Run**
+
+In `image_tools` ROS2 package, 2 executables are provided, namely `cam2image` and `showimage` with different functions.
+
+## **1 - cam2image**
+Running this executable connects to your workstation's default camera device's video stream and publishes on ROS2 publisher, `/image` and `/flipimage`.
```

Comment:
"... and publishes the images on '/image' and '/flipimage' topics using a ROS2 publisher."

Reply:
Thank you for the prompt review. Will edit accordingly.